frontend/libs/studio-components-legacy/src/components/StudioDecimalInput/StudioDecimalInput.tsx

```diff
@@ -41,6 +49,14 @@ export const StudioDecimalInput = forwardRef(
       [setInputValue, onChange],
     );
 
+    const handleInputBlur = useCallback(
+      (e: React.FocusEvent<HTMLInputElement>) => {
+        const input = e.target.value;
+        if (isStringValidDecimalNumber(input)) onBlurNumber?.(convertStringToNumber(input));
```

Comment:
Gitt at du gjør endringen jeg foreslo over, bør du også kalle `onBlur` her. Da sikrer vi at både `onBlur` og `onBlurNumber` fungerer slik de skal. Lag gjerne en test for `onBlur` også.
```suggestion
        const input = e.target.value;
        if (isStringValidDecimalNumber(input)) onBlurNumber?.(convertStringToNumber(input));
        onBlur?.(e);
```